index.js

```diff
@@ -8,18 +8,88 @@ module.exports = {
             schema: {
                 $schema: 'http://json-schema.org/draft-07/schema#',
                 type: 'object',
+                required: ['q-applicant-declaration'],
                 additionalProperties: false,
                 properties: {
                     'applicant-declaration': {
                         title: 'Declaration',
                         description:
                             '<p class="govuk-body">By submitting the application I, ||/answers/p-applicant-enter-your-name/q-applicant-title|| ||/answers/p-applicant-enter-your-name/q-applicant-first-name|| ||/answers/p-applicant-enter-your-name/q-applicant-last-name||, agree that:</p><ul class="govuk-list govuk-list--bullet"><li>the information I’ve given here is true as far as I know</li><li>CICA can share the information I’ve given in this claim with:</li><ul><li>police, prosecutors and ACRO Criminal Records Office</li><li>medical organisations and staff, including police medical staff</li><li>any other individuals or organisations needed to process my application (including medical or other experts)</li></ul><li>CICA can receive information from the organisations and individuals described above</li><li>If I deliberately provide information that I know is false or misleading, I may be prosecuted and my application for compensation may be refused.</li></ul><p class="govuk-body">We often have to ask your GP or other health service provider for evidence about your injuries and treatment. We will let you know if we need to do this.</p><p class="govuk-body">Read our privacy notice to see <a class="govuk-link" href="https://www.gov.uk/guidance/cica-privacy-notice">how we use your data</a>.</p>'
+                    },
+                    'q-applicant-declaration': {
+                        title: 'Do you agree to this declaration?',
+                        type: 'array',
+                        items: {
+                            anyOf: [
+                                {
+                                    title: 'Yes',
+                                    const: true
+                                }
+                            ]
+                        }
                     }
```

Comment:
This checkbox is a bit different to what's on the prototype. Also consider tightening the validation here, eg

```
    q-applicant-declaration': {
        title: 'Do you agree to this declaration?',
        type: 'array',
        maxItems: 1,
        uniqueItems: true,
        items: {
            anyOf: [
                {
                    title: 'Yes',
                    const: true
                }
            ]
        }
    }
```

Reply:
Yeah I have not even looked at the prototype yet. hence why it is in draft. will be changing that this afternoon. Just wanted to make you aware of this PR to iron out any huge issues